build-requirements.txt

```diff
@@ -1,6 +1,6 @@
 build==0.3.0 --hash=sha256:3fe8fac398ae76f534dee92c0db01c2960e271094f8cd52aa9d24a633c244e59
 click==7.1.2 --hash=sha256:3b0769bd65ca00ffcb57975e750fac079639a3d3ce393c5a0d4c65123141c87a
-cython==0.29.22 --hash=sha256:8c9eec7e9de2a30861ca347d0a149cc1482de12fc765fa06c414930e8ce20d0a
+cython==0.29.22 --hash=sha256:8c9eec7e9de2a30861ca347d0a149cc1482de12fc765fa06c414930e8ce20d0a --hash=sha256:df6b83c7a6d1d967ea89a2903e4a931377634a297459652e4551734c48195406
```

Comment:
did this already get diff-reviewed?

Reply:
It's the hash of the tarball that's already in the repository, so I assume it was reviewed.

Reply:
ah, i see that now!